src/amo/components/RatingManager/index.js

```diff
@@ -18,34 +21,52 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import DefaultRating from 'ui/components/Rating';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { UserReviewType } from 'amo/actions/reviews';
+import type { SubmitReviewParams } from 'amo/api';
+import type { UrlFormatParams } from 'core/api';
+import type { ApiStateType } from 'core/reducers/api';
+import type { DispatchFn } from 'core/types/reduxTypes';
+import type { AddonType, AddonTypeProp, AddonVersionType }
```

Comment:
Our check only enforces line length. I prefer it like this since the the import list fits all on one line but I can change it to your example, no biggie.